src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2863,15 +2863,6 @@ private void RemoveDirectoryInfoItem(DirectoryInfo directory, bool recurse, bool
                 continueRemoval = ShouldProcess(directory.FullName, action);
             }
 
-            //if this is a reparse point and force is not specified then warn user but dont remove the directory.
```

Comment:
Why is this removed.  It seems like the -Force parameter was required before removing and was Windows only.  So I assume removing this requirement makes Windows and Linux behavior the same.  Please make this clear in the PR description.

Also if we remove this code we should also remove the associated FileSystemProviderStrings.DirectoryReparsePoint message string since it is no longer used.